tembo-operator/src/dedicated_networking.rs

```diff
@@ -356,10 +356,16 @@ async fn reconcile_dedicated_networking_service(
         service_name, namespace, service_type, lb_scheme
     );
 
+    let external_dns_hostname = if is_standby {
+        format!("dedicated-ro.{}.{}", namespace, basedomain)
+    } else {
+        format!("dedicated.{}.{}", namespace, basedomain)
+    };
```

Comment:
Alternatively you could do something like this:

```rust
    let external_dns_hostname = format!(
        "dedicated{prefix}.{namespace}.{basedomain}",
        prefix = if is_standby { "-ro" } else { "" }
    );
```

Whichever you prefer

Reply:
Oh nice! Thank you